source/cli/instance-scheduler-cli-runner.py

```diff
@@ -15,7 +15,7 @@
 
 """Convenience wrapper for running instance-scheduler-cli directly."""
 
-from scheduler_cli.scheduler_cli import main
+from instance_scheduler_cli.scheduler_cli import main
 
 if __name__ == "__main__":
     main()
```

Comment:
outside the scope of this review: surely this can be eliminated by adding something to `init`...

Reply:
probably. I don't currently know enough of the nuance of python CLIs to be able to reason about why this should or should not exist